compute/admin_guide/configure/custom_feeds.adoc

```diff
@@ -78,7 +78,9 @@ It is also used immediately by the runtime defense file system sensor, which ass
 [.task]
 === Create a list of trusted executables
 
-If a legitimate binary or a process created from a legitimate binary is incorrectly identified as malicious by any of the runtime detection capabilities, you can add the signature of the binary to a list of trusted executables signatures and ensure that it will not be inspected by any runtime capabilities. 
+The Trusted Executable Feed is a mechanism to address potential misidentification of legitimate files as malware. You can add to the feed signatures of binaries that have been incorrectly categorized as malicious by the file system based malware runtime protection. 
```

Comment:
Hey @omri86 a couple questions:

1. is there a feed by this name "Trusted Executable Feed"? 
or are you saying that "you can create a custom feed that includes a list of trusted executables to"

2. Can we revise it to say 

If you have a list of trusted executables, you can add the signature of the binaries to a custom feed and create a trustable executables feed for your organization. This feed enables you to ensure that  a legitimate binary or a process created from a legitimate binary is not potentially identified as malicious by the file system based malware runtime protection.

Reply:
1.Yes, navigate to `system -> custom feeds -> trusted executables`
2. I don't think the revised sentence explains much better than what I wrote. It only adds confusion, because starting off with "If you have a list of trusted executables" does not explain to the customer what are those "trusted executables".

Reply:
@Pubs-MV just making sure you saw my comment

Reply:
Can you please review @omri86